lib/log_manager.py

```diff
@@ -3,22 +3,37 @@
 import os
 from datetime import datetime
 from collections import defaultdict
+from influxdb_client import InfluxDBClient
 
 
 class LogManager(object):
     def __init__(self, findmy_files, store_keys, timestamp_key, log_folder,
                  name_keys, name_separator, json_layer_separator, null_str,
-                 date_format, no_date_folder):
+                 date_format, no_date_folder, log_location, influx_host,
+                 influx_token, influx_org, influx_bucket):
         self._findmy_files = findmy_files
         self._store_keys = store_keys
         self._timestamp_key = timestamp_key
-        self._log_folder = log_folder
         self._name_keys = name_keys
         self._name_separator = name_separator
         self._json_layer_separator = json_layer_separator
         self._null_str = null_str
         self._date_format = date_format
-        self._no_date_folder = no_date_folder
+
+        # Log location config
+        self._log_location = log_location
+        if self._log_location == 'local':
+            self._log_folder = log_folder
+            self._no_date_folder = no_date_folder
+        elif self._log_location == 'influx':
+            self._influx_org = influx_org
+            self._influx_bucket = influx_bucket
+            influx_client = InfluxDBClient(
+                url=influx_host,
+                token=influx_token,
+                org=self._influx_org
+            )
+            self._influx_write_api = influx_client.write_api()
```

Comment:
Thanks so much for the nice feature! Could you add an else statement here to raise an error when self._log_location is neither local/influx?

Reply:
Updated to raise a ValueError on an unsupported log location